src/Controllers/Frontend.php

```diff
@@ -437,14 +385,31 @@ public static function template(Silex\Application $app, $template)
             $template .= '.twig';
         }
 
-        $themePath    = realpath($app['paths']['themepath'] . '/');
-        $templatePath = realpath($app['paths']['themepath'] . '/' . $template);
+        return self::render($app, $template, $template);
+    }
 
-        // Verify that the resulting template path is located in the theme directory
-        if ($themePath !== substr($templatePath, 0, strlen($themePath))) {
-            throw new \Exception("Invalid template: $template");
```

Comment:
You may notice that this check is gone, I don’t believe it is necessary to limit to current theme, especially if we want to support parent themes or having extensions adding custom templates. It could be argued that a check could be added to at least be in the themes directory.